src/libZilliqa/Zilliqa.cpp

```diff
@@ -97,7 +98,7 @@ Zilliqa::Zilliqa(const std::pair<PrivKey, PubKey>& key, const Peer& peer,
     , m_ds(m_mediator)
     , m_lookup(m_mediator)
     , m_n(m_mediator, syncType, toRetrieveHistory)
-    , m_cu(key, peer)
+    //    , m_cu(key, peer)
```

Comment:
Why preserve m_cu? For future possible usage?

Reply:
Nope, more like just to remember we have to do some clean-up work. Anyway this will be taken care of in #571 hopefully by next week.

Reply:
Got it, thanks.